fcc-test-suite-template-master/index.html

```diff
@@ -86,10 +86,6 @@ <h1><center>Here's a time line of Dr. César's life:</center></h1>
             <strong>1983</strong> - helps seven African countries dramatically
             increase their maize and sorghum yields
           </li>
-          <li>
-            <strong>1984</strong> - becomes a distinguished professor at Texas
-            A&amp;M University
-          </li>
           <li>
             <strong>2005</strong> - states "we will have to double the world
             food supply by 2050." Argues that genetically modified crops are the
```

Comment:
please make your code dynamic, use js

Reply:
- [ ] Kindly add other comments that are relevant to the changes you are asking for the student 🎯 you can look for another piece of code in your project that needs improvements and add your comments about it 👍🏼